src/runtime/StaticStrings.h

```diff
@@ -466,6 +470,7 @@ namespace Escargot {
     F(trimLeft)                   \
     F(trimRight)                  \
     F(trimStart)                  \
+    F(true_)                      \
```

Comment:
`true` string is also defined as `stringTrue`.

Reply:
Good suggestion, I haven't noticed it!
What do you think about this enhancement in general? Is this something you prefer or not?